docker/conf/node2/config/genesis.json

```diff
@@ -0,0 +1,35 @@
+{
+  "genesis_time": "2021-04-08T09:27:31.314819Z",
+  "chain_id": "test-chain-hHNinh",
```

Comment:
Should the 3 nodes share the same genesis document so that they work on the same chain?